docs/source/install/deb.rst

```diff
@@ -80,20 +80,21 @@ For remote linux actions, SSH is used. It is advised to configure identity file
   .. code-block:: bash
 
     # Create an SSH system user (default `stanley` user may be already created)
-    useradd stanley
-    mkdir -p /home/stanley/.ssh
-    chmod 0700 /home/stanley/.ssh
+    sudo useradd stanley
+    sudo mkdir -p /home/stanley/.ssh
+    sudo chmod 0700 /home/stanley/.ssh
 
     # On StackStorm host, generate ssh keys
-    ssh-keygen -f /home/stanley/.ssh/stanley_rsa -P ""
+    sudo ssh-keygen -f /home/stanley/.ssh/stanley_rsa -P ""
 
     # Authorize key-base acces
-    cat /home/stanley/.ssh/stanley_rsa.pub >> /home/stanley/.ssh/authorized_keys
-    chmod 0600 /home/stanley/.ssh/authorized_keys
-    chown -R stanley:stanley /home/stanley
+    sudo sh -c 'cat /home/stanley/.ssh/stanley_rsa.pub >> /home/stanley/.ssh/authorized_keys'
+    sudo chmod 0600 /home/stanley/.ssh/authorized_keys
+    sudo chown -R stanley:stanley /home/stanley
 
     # Enable passwordless sudo
-    echo "stanley    ALL=(ALL)       NOPASSWD: SETENV: ALL" >> /etc/sudoers.d/st2
+    sudo sh -c 'echo "stanley    ALL=(ALL)       NOPASSWD: SETENV: ALL" >> /etc/sudoers.d/st2'
```

Comment:
@dzimine it should be documented but not suggested as an action.
please review https://github.com/StackStorm/st2-packages/pull/195 . Since @enykeev @armab and me I +100 for this. We have to permissions story sorted out and documented, no we have none of that except those vague steps which are clear only for us.

1. actionrunner should operate as `system_user`, to avoid excessive privilege elevation https://github.com/StackStorm/st2/blob/master/st2common/st2common/models/system/action.py#L79-L93 . Everytime when we are `!= system_user` we use sudo, that doesn't make sense.
2. we need permission allowance cross st2api and st2actionrunner to enable flow to save packs.

The current not documented approach how to run and how privileges are elevated is only a mess and a trouble.